### ISSUE TITLE
Gnupg 2.1 breaks  python-gnupg

### ISSUE THREAD

**Issue by keisetsu** [SexualHealthInnovations/callisto-core#115]
There seem to be two main versions of gnupg: 2.0 and 2.1. According to the website, 2.0.30 is stable, while 2.1 is modern (?). I guess there are some breaking changes in 2.1, and I think most Linux distros stick with 2.0, though the one I'm using, Arch, requires a downgrade.

In any case, while this is a known issue on the python-gnupg upstream github list ([here, for example](https://github.com/isislovecruft/python-gnupg/issues/143)), the issues haven't been addressed and it doesn't look like there has been a commit to master since March of 2015 (one branch had a commit in March of this year, but still...).

Long story short, for now, developers and users may have to downgrade to 2.0 if their OS uses 2.1. But it looks like Python support for GnuPG is a bit shaky in the long term, unless someone takes over development of python-gnupg.

**Comment by lynnco** [comment]
@keisetsu apologies for asking you for labor again! but can you remember if https://github.com/isislovecruft/python-gnupg/issues/143 resolved this issue?

**Comment by keisetsu** [comment]
Sorry, Lynn, I didn't have a look at it - I passed it on to Kelsey.

**Comment by lynnco** [comment]
@keisetsu gotcha, thanks!

**Comment by lynnco** [comment]
Looks like we switched from that gnupg package in this PR https://github.com/SexualHealthInnovations/callisto-core/pull/104

**Comment by lynnco** [comment]
But both https://pypi.python.org/pypi/gnupg and https://pypi.python.org/pypi/python-gnupg are broken for me on local right now, with a gnupg version `2.1.21`.

I'm looking into switching (back) https://github.com/isislovecruft/python-gnupg, its giving a more easily readable status message and I found the open issue for the error I have on local https://github.com/isislovecruft/python-gnupg/issues/157

**Comment by lynnco** [comment]
It looks like basically most of the gpg tooling ecosystem doesn't fully support gpg 2.1, so here's the downgrade instructions for osx

```bash
brew uninstall -f gnupg
brew install gnupg@2.0
```
